core-bundle/assets/controllers/sortable-controller.js

```diff
@@ -36,6 +36,12 @@ export default class extends Controller {
 
             if (handles.length === 1) {
                 handles[0].style.display = '';
+            } else {
+                for (const handle of handles) {
+                    if (handle.style.display === 'none' && handle.parentNode.localName === 'li') {
+                        handle.parentNode.style = 'display: none !important';
```

Comment:
What does this do?

Reply:
The changes in https://github.com/contao/contao/pull/8372/files#diff-5b5d76cbcb59ea85b5592432c3dfb870e5e507447a5bbbca8ce69f255e5c408d (also allowing `<button>` in operations menu now shows an empty `<li>` in the operation menu if the parent view has a drag handle. This hides the `<li>` (similar to https://github.com/contao/contao/blob/5.x/core-bundle/assets/controllers/limit-height-controller.js#L140)

Reply:
That doesn't seem to work:

<img width="620" alt="" src="https://github.com/user-attachments/assets/d577c133-8a9d-43c9-858c-c3de0662b375" />

Reply:
We should never show drag handles in the context menu IMHO.